packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.tsx

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { usePageViewEvent } from '../../../lib/metrics';
+
+export const PageRecoveryKeyCreate = (props: RouteComponentProps) => {
+  const [currentStep, setCurrentStep] = useState(1);
+  usePageViewEvent('settings.account-recovery');
+
+  /*
+    The content here will obviously be replaced as we complete the separate views for this flow. This page will use the same pattern as the example wizard in storybook for `FlowContainer`.
+  */
+  return (
+    <>
+      {
+        // Create an account recovery key
+        currentStep === 1 && (
+          <>
+            <p>first step</p>
```

Comment:
Should this have a comment to indicate all steps need to use `FlowContainer`? Feel free to ignore me since we both know we'll need that component :)

Reply:
I omitted it since the comment above mentions to reference the FlowContainer storybook, and that has clearer examples, but happy to change!